hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -54,13 +54,23 @@
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientMapBasicTest {
+    private static final String KEY = "key";
+    private static final String OLD_VALUE = "oldValue";
+    private static final String NEW_VALUE = "newValue";
 
-    static HazelcastInstance client;
-    static HazelcastInstance server;
+    private static HazelcastInstance client;
+    private static HazelcastInstance server1;
+    private static HazelcastInstance server2;
+    private static HazelcastInstance server3;
+
+    private String randomMapName = randomString();
+    private IMap<String, String> randomStringMap = client.getMap(randomMapName);
 
```

Comment:
The `identityHashCode()` is different, just checked that. Everything else would be very surprising, since I know it as best practice to use non-static fields in tests for data which is created identically (but not shared between) tests.